controller/appcontroller.go

```diff
@@ -1111,38 +1114,68 @@ func (ctrl *ApplicationController) finalizeApplicationDeletion(app *appv1.Applic
 		}
 		if len(objsMap) > 0 {
 			logCtx.Infof("%d objects remaining for deletion", len(objsMap))
-			return objs, nil
+			return nil
 		}
+		logCtx.Infof("Successfully deleted %d resources", len(objs))
+		logCtx.Infof("Resource entries removed from undefined cluster")
```

Comment:
Is this log needed here? If it is needed not sure if it should be `undefined cluster`

Reply:
Ops, thank you for catching it. I just forgot to delete this line - fixed!